core/src/main/java/com/sequenceiq/cloudbreak/service/LoadBalancerUpdateService.java

```diff
@@ -0,0 +1,15 @@
+package com.sequenceiq.cloudbreak.service;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.dto.NameOrCrn;
+
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.flow.api.model.FlowIdentifier;
+
+@Service
+public class LoadBalancerUpdateService {
+
+    public FlowIdentifier updateLoadBalancers(NameOrCrn nameOrCrn, Long workspaceId) {
+        return null;
```

Comment:
why is this null?

Reply:
Because this PR is intended just to have the API changes and to stub out the services they call. The actual implementation of those services are in PRs 9963 and 9964. I broke the change into three PRs because it's a large change, and I thought that would be easier for people to review.

Reply:
Updated the description to make this more clear.